### PR TITLE
Replaced Danswer to Darwin in frontend

### PR DESCRIPTION
**Pull request by shreyash0502**

### CODE REVIEW

File: web/src/app/admin/bot/SlackBotConfigCreationForm.tsx [hunk 1]
```diff
@@ -175,10 +175,10 @@ export const SlackBotCreationForm = ({
                     <div>
                       The names of the Slack channels you want this
                       configuration to apply to. For example,
-                      &apos;#ask-danswer&apos;.
+                      &apos;#ask-darwin&apos;.
```
Comment (vivekjhauipath) [inline web/src/app/admin/bot/SlackBotConfigCreationForm.tsx:178]: #darwin-feedback

File: web/src/app/admin/bot/SlackBotConfigCreationForm.tsx [hunk 2]
```diff
@@ -155,8 +155,8 @@ export const SlackBotCreationForm = ({
               const errorMsg = responseJson.detail || responseJson.message;
               setPopup({
                 message: isUpdate
-                  ? `Error updating DanswerBot config - ${errorMsg}`
-                  : `Error creating DanswerBot config - ${errorMsg}`,
+                  ? `Error updating DarwinBot config - ${errorMsg}`
+                  : `Error creating DarwinBot config - ${errorMsg}`,
```
Comment (vivekjhauipath) [inline web/src/app/admin/bot/SlackBotConfigCreationForm.tsx:159]: Darwin instead of DarwinBot everywhere

**Review by vivekjhauipath — APPROVED** [review]
LGTM

File: web/src/app/admin/bot/SlackBotConfigCreationForm.tsx (hunk 1 above)
Comment (shreyash0502) [reply web/src/app/admin/bot/SlackBotConfigCreationForm.tsx:178]: done

File: web/src/app/admin/bot/SlackBotConfigCreationForm.tsx (hunk 2 above)
Comment (shreyash0502) [reply web/src/app/admin/bot/SlackBotConfigCreationForm.tsx:159]: done

**Pull request merged**